react/features/welcome/components/WelcomePageSideBar.native.js

```diff
@@ -0,0 +1,167 @@
+// @flow
+
+import React, { Component } from 'react';
+import { SafeAreaView, ScrollView, Text } from 'react-native';
+import { connect } from 'react-redux';
+
+import SideBarItem from './SideBarItem';
+import styles from './styles';
+
+import { setSideBarVisibility } from '../actions';
+
+import { showAppSettings } from '../../app-settings';
+import {
+    Avatar,
+    getAvatarURL,
+    getLocalParticipant,
+    getParticipantDisplayName
+} from '../../base/participants';
+import {
+    Header,
+    SideBar
+} from '../../base/react';
+
+/**
+ * The URL at which the privacy policy is available to the user.
+ */
+const PRIVACY_URL = 'https://jitsi.org/meet/privacy';
+
+/**
+ * The URL at which the user may send feedback.
+ */
+const SEND_FEEDBACK_URL = 'mailto:support@jitsi.org';
+
+/**
+ * The URL at which the terms (of service/use) are available to the user.
+ */
+const TERMS_URL = 'https://jitsi.org/meet/terms';
+
+type Props = {
+
+    /**
+     * Redux dispatch action
+     */
+    dispatch: Function,
+
+    /**
+     * The avatar URL to be rendered.
+     */
+    _avatar: string,
```

Comment:
You're right, it would be more consistent. This is also in the (older) welcome screen facelift PR (which the calendar sync PR is rebased to), so I think I'll create a new PR a bit later, addressing some of your points here